### DISCUSSION TITLE
Pre-announcement new version of tau-argus

### DISCUSSION THREAD

**Discussion by ppdewolf** [sdcTools/UserSupport#278] [Team Testers]
As discussed during one of the online Uer Group meetings I would inform you of any intentions of new releases a bit ahead of the release date.
Well: I intend to upload a new version of tau-argus (version 4.2.5) before the end of May 2024. The exact date will be announced in this discussion when known.

The main new features to be tested:

- `Select Table` should now work properly (issue #277) 
- Improved speed in reading `hst`-files
- Scrollbar available in selection pane in SelectTableMicroDialog (issue #184) 
- Fixed issue #267
- Parameter calculation GHMIter for Large Model corrected, now consistent with general model

Please also check the linked approaches (modular and hypercube) as there have been some internal code changes not related to any specific issue or new feature.

And, as always, please check your existing set-up, tables, batch-files, etc. whether they still work as before ;-)

**Comment by julienjamme** [comment]
Hi Peter-Paul,  We are looking forward to testing this new version, especially the improved speed of hst files.

**Comment by ppdewolf** [comment]
The test version of tau-argus 4.2.5 is now available for download: https://github.com/sdcTools/tauargus/releases/tag/v4.2.5-TEST 
Please test this version and report any findings before 8 July 2024.

**Comment by julienjamme** [comment]
Hi Peter-Paul,

Here are my first feedbacks about tau-argus 4.2.5:

- Speed in reading apriori files is really improved with this new version. I've tested it, especially, on a file that took 3 days to read the apriori file with the 4.2.3. Now it's two minutes max. So thank you for that.
- I've checked whether the masks on some of our productions (several linked tables) remain the same when moving from 4.2.3 to 4.2.5. I called tau-argus from R with rtauargus. And, the masks are indeed identical.  Only "Modular" has been tested but I can confirm for Hypercube later on the same examples.

**Reply by ppdewolf** [reply]
Thank you for the feedback!
